sw/device/silicon_creator/mask_rom/mask_rom_start.S

```diff
@@ -162,13 +162,16 @@ _mask_rom_start_boot:
   // The following sequence enables the minimum level of entropy required to
   // initialize memory scrambling, as well as the entropy distribution network.
   li a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
-  li t0, (0xa << ENTROPY_SRC_CONF_ENABLE_OFFSET) | \
+  li t0, (0xa << ENTROPY_SRC_CONF_FIPS_ENABLE_OFFSET) | \
```

Comment:
I added #10259 to track future changes to the initialization sequence in the mask ROM.